### ISSUE TITLE
Header should be H2

### ISSUE THREAD

**Issue by akeller** [camunda/camunda-platform-docs#2538]
https://github.com/camunda/camunda-platform-docs/blob/365ad853be58ec73dc4ba35dc3c7392a0768ded7/src/pages/meta.md?plain=1#L81

Whoops! Looks like this is missing an `#`. Without it, we can't get a link and this is a violation of accessibility guidelines (too many H1s per page - should only be 1).

**Comment by christinaausley** [comment]
Quick win for the day 👍 https://github.com/camunda/camunda-platform-docs/pull/2539
